bus-mapping/src/circuit_input_builder.rs

```diff
@@ -550,15 +561,26 @@ impl<P: JsonRpcClient> BuilderClient<P> {
     pub async fn gen_inputs(
```

Comment:
gen_inputs will be used by zkevm-chain. Please see the PR: https://github.com/smtmfft/zkevm-chain/pull/2